sirepo/package_data/static/js/radia.js

```diff
@@ -169,6 +179,13 @@ SIREPO.app.factory('radiaService', function(appState, fileUpload, panelState, re
         self.showPathPicker(true, true);
     };
 
+    self.nextId = function() {
```

Comment:
This code seems particularly confusing. I think a comment saying why we need a uuid of this shape would be helpful.

Reply:
uuid is appropriate so I'll document the reasoning (```nextId()``` is historical)